subprojects/docs/src/samples/userguideOutput/incrementalBuildInputFilesConfig.out

```diff
@@ -1,4 +1,4 @@
-:clean
+:clean UP-TO-DATE
```

Comment:
This is related to `delete.includeEmptyDirs = true`. We didn't delete the directory before, only the contents of the directory. The sample is executed multiple times via `UserGuideSamplesRunner`.